test/WebJobs.Script.Tests/TestScripts/Node/NextTick/function.json

```diff
@@ -0,0 +1,9 @@
+﻿{
+    "bindings": [
```

Comment:
Can you just add this as a new branch in the Scenarios function? See the pattern I've been employing. You can then add a test that does a manual invoke to make sure it doesn't hang.